### ISSUE TITLE
handle UTF-8 chars

### ISSUE THREAD

**Issue by VladimirAlexiev** [RDFLib/pyLODE#110]
I added owl:Ontology and rdfs:isDefinedBy to schema.org (#108).

Now `pylode -i schema-with-added-ontology.ttl -c true -o schema.html`
fails with
`'charmap' codec can't encode character '\xa3' in position 654973: character maps to <undefined>`

It's on the pound sign in this comment: `balance between £1,000 and £1,000,000 GBP`.
The file is valid UTF-8 without BOM.

**Comment by VladimirAlexiev** [comment]
test files: https://gist.github.com/VladimirAlexiev/d2ac78006c8066d1230121db158eaf13/archive/b8c70c52a28513578198484ad7bcf86ea75b045d.zip

**Comment by nicholascar** [comment]
This issue seems to be fixed in the latest release, 2.8.6. I say "seems" because it appeared to be working already. A test has been created for this using the files given above, see https://github.com/RDFLib/pyLODE/blob/master/tests/test_issue_110.py.

**Comment by nicholascar** [comment]
Reopening as issue still present on Windows (I just didn't see it on my Mac)

**Comment by smarie** [comment]
I have a similar issue but PyLODE does not error - instead the generated html page contains an invalid character.

My source file is an UTF-8 turtle file with accentuated french character "é"

**Comment by smarie** [comment]
My bad I was using PyLODE 2.4... :)

**Comment by VladimirAlexiev** [comment]
As you can see in #119, `pip install pylode` doesn't upgrade from 2.8.5 to any later version.

So I got https://github.com/RDFLib/pyLODE/archive/refs/tags/2.8.6.zip and unzipped pylode.exe to `c:\Users\vladimir.alexiev.000\AppData\Local\Programs\Python\Python37\Scripts\`.

Now it works:
```
pylode -i schema-with-added-ontology.ttl -c true -o schema.html
Finished. ontdoc documentation in schema.html
```

and produces this:
`or a range as in describing a bank account being suitable for a balance between £1,000 and £1,000,000 GBP, or the value of a salary, etc. It is recommended to use [[PriceSpecification]] Types to describe the price of an Offer, Invoice, etc.`

**Comment by smrgeoinfo** [comment]
note this fix apparently didn't get into the 3.0.1 build...
